enterprise/backend/src/metabase_enterprise/gsheets/settings.clj

```diff
@@ -65,7 +65,7 @@
   (deferred-tru "Information about Google Sheets Integration")
   :encryption :when-encryption-key-set
   :export? true
-  :visibility :public
+  :visibility :admin
```

Comment:
This setting is only used for admins, and it still is returned by api/session/properties with this visibility setting.